modules/client-configuration/pages/supported-features-centos.adoc

```diff
@@ -15,59 +15,215 @@ The operating system you run on a client is supported by the organization that s
 
 The icons in this table indicate:
 
-* icon:check[role="green"] the feature is available on both traditional and Salt clients
-* icon:times[role="danger"] the feature is not available
-* icon:question[role="gray"] the feature is under consideration, and may or may not be made available at a later date
-* ``Traditional`` the feature is supported only on traditional clients
-* ``Salt`` the feature is supported only on Salt clients.
+* {check} the feature is available on both traditional and Salt clients
```

Comment:
Wondering whether we'd better mention Salt before traditional.

Reply:
Sure :+1: